naff/models/naff/checks.py

```diff
@@ -72,7 +72,7 @@ def is_owner() -> TYPE_CHECK_FUNCTION:
     async def check(ctx: Context) -> bool:
         if ctx.bot.app.team:
             return ctx.bot.app.team.is_in_team(ctx.author.id)
```

Comment:
This currently ignores owner_ids if the bot is in a team, which feels wrong.  We should instead add the entire team to owner_ids, and then always check that.

Reply:
Resolved in 7bf04825333e1d4b0a86d3c57d528bd1c6fed023